packages/core/src/delegation/DelegationRootNode.spec.ts

```diff
@@ -5,30 +5,37 @@
  */
 
 import { Crypto } from '@kiltprotocol/utils'
+import {
+  BlockchainApiConnection,
+  BlockchainUtils,
+} from '@kiltprotocol/chain-helpers'
+import { mockChainQueryReturn } from '@kiltprotocol/chain-helpers/lib/blockchainApiConnection/__mocks__/BlockchainQuery'
 import { Identity } from '..'
-import { BlockchainUtils } from '../blockchain'
-import getCached from '../blockchainApiConnection'
-import { mockChainQueryReturn } from '../blockchainApiConnection/__mocks__/BlockchainQuery'
+
 import DelegationRootNode from './DelegationRootNode'
+import Kilt from '../kilt/Kilt'
 
-jest.mock('../blockchainApiConnection/BlockchainApiConnection')
+jest.mock(
+  '@kiltprotocol/chain-helpers/lib/blockchainApiConnection/BlockchainApiConnection'
+)
```

Comment:
I'm wondering whether package-based auto-mocks would work here, e.g. linking to a mock in a folder structure that sits alongside the current packages `node-modules` folder (see [here](https://jestjs.io/docs/en/manual-mocks#mocking-node-modules)).

Reply:
Will be handled with https://github.com/KILTprotocol/ticket/issues/1026